hack/test/local_test.sh

```diff
@@ -66,23 +66,24 @@ if [[ ! $(pwd) =~ .*certman-operator$ ]]; then
   exit
 fi
 
-echo "Checking if docker service is active"
-systemctl is-active docker
+#echo "Checking if docker service is active"
```

Comment:
Can we support both podman and docker in this script?

Reply:
that should be added now